extra/utils/helpers.js

```diff
@@ -0,0 +1,11 @@
+export function validateEmail(email){
+    var reg = /^([A-Za-z0-9_\-\.])+\@([A-Za-z0-9_\-\.])+\.([A-Za-z]{2,4})$/;
```

Comment:
this regex is a common email regex please create a custom regexp for the successive tech mail id